### ISSUE TITLE
"Capture Visible" keyboard shortcut (ctrl+shift+v) conflicts with important shortcut

### ISSUE THREAD

**Issue by voxadam** [rojer/a-s-minus#15]
Awesome Screenshot Minus' default keyboard shortcut for capturing the visible portion of a page conflicts with Chrome's shortcut for pasting plain text (sans the formatting of the source). The shortcut is not only used in Chrome but has become somewhat of a de facto standard in recent years.

**Comment by rojer** [comment]
yes, this is true. in fact, i have "capture visible" shortcut reconfigured to c-s-w because i use c-s-v a lot myself.
this is something i inherited from upstream. maybe it's time to change the default.

**Comment by voxadam** [comment]
I just ran into this little issue again. Have you given any further thought to a new mapping?

**Comment by Korfys** [comment]
Looks like either some people overlooked the ability to change the shortcut keys, or that the plugin has been updated to include that ability.  Either way, this doesn't seem to be an issue any more - except for perhaps packaging the plugin with a different default.
